addons/hr_holidays/static/tests/new/im_status_tests.js

```diff
@@ -57,5 +57,5 @@ QUnit.test("on leave & offline", async function (assert) {
     });
     await openDiscuss(channelId);
     await afterNextRender(() => advanceTime(UPDATE_BUS_PRESENCE_DELAY));
-    assert.containsOnce(document.body, ".o-mail-im-status i.fa-plane[title='Out of office']");
+    assert.containsOnce($, ".o-mail-im-status i.fa-plane[title='Out of office']");
```

Comment:
I really don't like this. This will result in the target being `$(document)` which could interfere with tests. I remember a test that was only failing when another one failed. This was due to an assertion being made on a `containsOnce(document.body, "....")` that was matching a `QUnit` element. I think asserting on `target` as we do in many tests is better as is restrict the assertion on the webclient realm.